### PR TITLE
Externalizing Cassandra and Coordinator timeout values for test container

### PR DESCRIPTION
**Pull request by maheshrajamani**
Default value will remain the same as it is now.

**Checklist**
- [ x] Changes manually tested
- [ ] Automated Tests added/updated
- [ ] Documentation added/updated
- [ ] CLA Signed: [DataStax CLA](https://cla.datastax.com/)

### CODE REVIEW

File: apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/common/testresource/StargateTestResource.java [hunk 1]
```diff
@@ -81,6 +81,9 @@ interface Defaults {
     String CLUSTER_VERSION = "4.0";
 
     String CLUSTER_DSE = null;
+
+    long CASSANDRA_STARTUP_TIMEOUT = 2;
```
Comment (tatu-at-datastax) [inline apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/common/testresource/StargateTestResource.java:85]: These are in minutes? Would make sense to add a comment mentioning this?

**Review by tatu-at-datastax — APPROVED** [review]
Makes sense, but I think this should be documented somewhere (if we have a README or something)? Maybe `sgv2-quarkus-common` needs `TESTING.md`.

File: apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/common/testresource/StargateTestResource.java (hunk 1 above)
Comment (tatu-at-datastax) [reply apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/common/testresource/StargateTestResource.java:85]: Actually, thinking about this some more: should we require it in minutes, or would it make more sense to use seconds value? To allow more granular configurability (90 seconds etc)

Comment (maheshrajamani) [reply apis/sgv2-quarkus-common/src/test/java/io/stargate/sgv2/common/testresource/StargateTestResource.java:91]: This is timeout value for starting up the service, so not sure if we need seconds level granularity. Added comments to for the defaults.

**Comment by maheshrajamani** [comment]
> Makes sense, but I think this should be documented somewhere (if we have a README or something)? Maybe `sgv2-quarkus-common` needs `TESTING.md`.

Yes we need these System property documented somewhere. Will create a issue for this and complete it.

**Comment by ivansenic** [comment]
@maheshrajamani +1 for the documentation.. Just don't forget to document the props from the `IntegrationTestUtils` those are far more important than these container ones.. Anyway a nice `TESTING.md` would be really helpful..

**Pull request merged**